src/itemdb.d

```diff
@@ -368,9 +370,14 @@ final class ItemDatabase
 					if (r2.empty) {
 						// root reached
 						assert(path.length >= 4);
-						// remove "root"
-						if (path.length >= 5) path = path[5 .. $];
-						else path = path[4 .. $];
+						// remove "root/" from path string if it exists
+						if (path.length >= 5) {
+							if (canFind(path, "root/")){
+								path = path[5 .. $];
+							}
```

Comment:
So this was the error with the disappearing strings it seems, as I suggested, right?

Reply:
Correct ... and depending on the 'input path' would truncate regardless what was there .. so had to update it to only truncate, if 'root/' was present to truncate.